plugins/generic/pdfJsViewer/templates/display.tpl

```diff
@@ -29,7 +29,7 @@
 
 		<a href="{$submissionUrl}" class="return">
 			<span class="pkp_screen_reader">
-				{translate key="catalog.viewableFile.return" monographTitle=$publishedSubmission->getLocalizedTitle()|escape}
+				{translate key="catalog.viewableFile.return" monographTitle=$publishedSubmission->getCurrentPublication()->getLocalizedTitle()|escape}
```

Comment:
If a PDF from an older publication would be displayed here, the title would not be correct. So maybe to provide 'publication' to the template in CatalogHandler::download() and use it here.